Aptoide/aptoide-client-v8
# [ASV-1340] task: remove unused code

**What does this PR do?**

   Remove unused social methods and related dependencies.
   Remove unused abtesting manager dependency from the appviewmanager provider.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] SocialRepository.java
- [ ] FragmentModule.java

**How should this be manually tested?**

  Compile successfully - I would advice to jump into the commit before the head and check the changes to be sure nothing used was removed.

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1340](https://aptoide.atlassian.net/browse/ASV-1340)

**Code Review Checklist**
